### ISSUE TITLE
[a11y]: Accessibility violation due to typo in Toggle button web component

### ISSUE THREAD

**Issue by szinta** [carbon-design-system/carbon#20106]
### Package

@carbon/web-components

### Package version

2.33.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker latest deployment

### Description

A [typo](https://github.com/carbon-design-system/carbon/blob/a10d9b3823cf01009b37966405a5eb0e1bb7caee/packages/web-components/src/components/toggle/toggle.ts#L149) in the `cds-toggle` component is triggering the below accessibility violation. 

<img width="1298" height="187" alt="Image" src="https://github.com/user-attachments/assets/52c883eb-67c7-431f-8667-2e10b2a2b53d" />

### Reproduction/example

https://web-components.carbondesignsystem.com/?path=/story/components-toggle--default

### Steps to reproduce

1. Open the Carbon Web Components Storybook.
2. Navigate to the default story of the Toggle button component.
3. Run an accessibility scan using the IBM Accessibility Checker browser extension.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
